hnn_core/tests/test_network.py

```diff
@@ -28,3 +29,35 @@ def test_network():
     for ev_input in params['t_ev*']:
         type_key = ev_input[2: -2] + ev_input[-1]
         assert len(net.gid_dict[type_key]) == net.N_cells
+
+
+def test_external_rhythmic_feeds():
```

Comment:
I would actually put this in a new file called `test_feed.py`

Reply:
done